pkg/i2gw/providers/gce/storage.go

```diff
@@ -17,16 +17,30 @@ limitations under the License.
 package gce
 
 import (
+	apiv1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	"k8s.io/apimachinery/pkg/types"
+	backendconfigv1 "k8s.io/ingress-gce/pkg/apis/backendconfig/v1"
+	backendconfigv1beta1 "k8s.io/ingress-gce/pkg/apis/backendconfig/v1beta1"
 )
 
 type storage struct {
 	Ingresses map[types.NamespacedName]*networkingv1.Ingress
+
+	// BackendConfig is a GKE Ingress extension, and it is associated to an GKE
+	// Ingress through specifying `cloud.google.com/backend-config` or
+	// `beta.cloud.google.com/backend-config` annotation on its Backend Services.
```

Comment:
I thought these annotations were actually equivalent, just that one happened to be newer and without `beta` in the name. I'm not sure which one has precedence, but that should be fairly easy to find in ingress-gce.

Reply:
They are both used to point to backend config, but we do need to do query on both annotations because this annotation is added by user in the yaml so we don't have control over which annotation they choose.
As a reference in ingress controller, here is how we are querying with both annotations https://github.com/kubernetes/ingress-gce/blob/master/pkg/annotations/service.go#L443